src/metiq.py

```diff
@@ -783,6 +986,23 @@ def __call__(self, parser, namespace, values, option_string=None):
         metavar="output-file",
         help="output file",
     )
+    parser.add_argument(
+        "-ao",
+        "--audio-offset",
+        type=float,
+        dest="audio_offset",
+        default=default_values["audio_offset"],
+        metavar="audio_offset",
+        help="Adjust problem in sync for either source of measuring device",
+    )
+    parser.add_argument(
+        "-ll",
```

Comment:
Please remove this one: metiq uses the traditional POSIX argument syntax convention.

https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html